### ISSUE TITLE
d4

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#1553]
rerolls itself sometimes

**Comment by ZMBanshee** [comment]
Possibly because of https://github.com/UnderscoreKilburn/repentance-issues/issues/1416 ?

**Comment by CoolG0101** [comment]
> Possibly because of #1416 ?

happened to me without backpack in-tact, however one interesting thing to note that the very first reroll I did with d4 (in boss rush) rerolled my backpack on my natural run. i wonder if it's related in any shape or form

https://clips.twitch.tv/CarefulDoubtfulWatermelonNinjaGrumpy--fOzHLQaz0yKp9Kg

**Comment by Lonrite** [comment]
> Possibly because of #1416 ?

happened to me without schoolbag in-tact, however one interesting thing to note that the very first reroll I did with d4 (in boss rush) rerolled my backpack on my natural run. i wonder if it's related in any shape or form

https://clips.twitch.tv/CarefulDoubtfulWatermelonNinjaGrumpy--fOzHLQaz0yKp9Kg

**Comment by ZMBanshee** [comment]
It happens on the roll after you lose Schoolbag, so the video you sent doesn't disprove that it was the cause

**Comment by Lonrite** [comment]
> It happens on the roll after you lose Schoolbag, so the video you sent doesn't disprove that it was the cause

yeah I know, what I was talking about is that D4 didn't instantly reroll itself after schoolbag, it was delayed after a few rolls so idk how consistent it is exactly
